app/src/main/java/app/attestation/auditor/AttestationActivity.java

```diff
@@ -507,6 +506,10 @@ public boolean onPrepareOptionsMenu(final Menu menu) {
         return true;
     }
 
+    final String tutorial_url() {
```

Comment:
Should make this `private` since Java defaults to package public. Don't need to make it `final` since if we want to do that it probably makes more sense to just make the classes `final` but it isn't really needed. It doesn't have the usual meaning of an immutable variable binding here but rather preventing an override by a class inheriting this one.